README.rst

```diff
@@ -0,0 +1,183 @@
+
+Fully Qualified Collection Name (FQCN)
+  The full definition of a module, plugin, or role hosted within a collection, in the form ``namespace.collection.content_name``. Allows a Playbook to refer to a specific module or plugin from a specific source in an unambiguous manner, for example, ``community.grafana.grafana_dashboard``. The FQCN is required when you want to specify the exact source of a module and multiple modules with the same name are available. Can always be identified in a playbook; ideally not necessary in most playbooks, but in cases in which users have multiple collections installed with similar content, the FQCN will always be the explicit and authoritative indicator of which collection to use for content. Example: cisco.ios.ios_config would be the FQCN, and the playbook would generally call "ios_config". When is this required.
+
+Namespace
+  The first part of a Fully Qualified Collection Name, the namespace usually reflects a functional content category. Example: in ``cisco.ios.ios_config``, “Cisco” is the Namespace. Namespaces are reserved and distributed by Red Hat at Red Hat’s discretion. Many, but not all, namespaces will correspond with vendor names. May be different from the GitHub organization name.
+
+Collection name
+  The second part of a Fully Qualified Collection Name, the collection name further divides the functional characteristics of the collection content and denotes ownership.  For example, the cisco namespace might contain  cisco.ios, ``cisco.ios_community``, and ``cisco.ios_prc``, containing content for managing ios network devices maintained by Cisco.
+
+The community.general collection
+  A special collection managed by the Ansible Community Team containing all the modules and plugins which shipped in Ansible 2.9 that don't have their own dedicated Collection. A work in progress can be found in `community.general <https://github.com/ansible-collection-migration/community.general/>`_ repository.
+
+Repository
+  The location of the source code included in a collection. Contributors make suggestions, fix bugs, and add features through the repository. Collection owners can host repositories on GitHub, Gerrit, or any other  source code repository platform they choose.
+ 
+ 
+Documentation
+==============
+
+* `Using Ansible Collections <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`_
+* `Using Developing Collections <https://docs.ansible.com/ansible/latest/dev_guide/developing_collections.html>`_
+ 
+Work needed
+===========
+
+Ansible 2.9 already contains basic support for Collections.
+
+The majority of the Ansible 2.10 release cycle is for:
+
+* Defining what the split of collections should be
+* Defining which modules go into these new modules
+* Defining ansible-base (ie which modules stay in ansible/ansible)
```

Comment:
Is this still up for discussion, or is it pretty much set in stone? I remember there was some analysis of module usage for 'core' modules, but I don't remember if that was the main criteria, or even if all the modules marked as 'core' would be staying in ansible-base.